### ISSUE TITLE
UI shows unneeded for replica/primary hosts probes as UNDEF

### ISSUE THREAD

**Issue by vvitad** [dalibo/temboard#1200]
Hello, again!

As I continue my way as DBA I came across another issue. 
I have: primary1, replica1, replica2. All monitored by temboard and connected through Patroni.
Yeasteday there was a switchover, and primary1 became replica1, and replica1 was promoted to primary1. In 10 minutes there was another switchover to the way it was before the frst one.
After that Temboard UI shows UNDEF:
1) for primary1
Streaming replication lag UNDEF
2) for replica1
WAL files ready to be archived, 
WAL files

Logs show no errors except this: WARNING:  monitoring: Unsupported PostgreSQL version for probe replication_connection, but I am aware of this. 

PostgreSQL v9.6
```
$ temboard-agent --version
temBoard agent 8.0 (/usr/bin/temboard-agent)
System Oracle Linux Server 7.6
Python 3.6.8 (/usr/bin/python3)
bottle 0.12.21
cryptography 2.3
libpq 9.6.15
psycopg2 2.8.5 (dt dec pq3 ext lo64)

$ temboard --version
temBoard 8.0 (/usr/bin/temboard)
System Oracle Linux Server 7.6
Python 3.6.8 (/usr/bin/python3)
cryptography 2.3
Tornado 4.5.3
libpq 9.6.15
psycopg2 2.8.5 (dt dec pq3 ext lo64)
SQLAlchemy 1.1.3

```

I have another cluster with the same configuration and there is no 'Streaming replication lag' for primary and no 'WAL files ready to be archived' and 'WAL files' for  replica. Thus I think temboard got somehow confused between those switchovers, and didn't get that the last switchover happend.

**Comment by vvitad** [comment]
Still hanging in there, looks like everyhing works fine, despite this weird bug. Still would love to figure it out

**Comment by vvitad** [comment]
Thank you a lot @bersace!
Could I ask something else.. Right now our primary node is being used for the temboard repository/server. What would be the best way of moving temboard server to another host?
I think about doing something like this:
1) install temboard on a new host, autoconfigure
2) stop current temboard server
3) take a dump of temboard database 
4) install temboard on a new host
5) than i guess I can change temboard-agent configs so it would point to our new server?
6) restore the dump on a new database?
7) delete temboard on the primary node

**Comment by vvitad** [comment]
> @vvitad for moving temboard, see #1203 .

oh.  so just move /etc/temboard and dump all data into the new repository.
many thanks

**Comment by vvitad** [comment]
@bersace hello!
so i moved the server to another host and moved the data with pg_dump
obviously my UNDEFS moved with the temboard.monitoring schema. it makes me think I could find the exact rows that cause it and delete them.. I am loking through code now, but I would be grateful if you pointed out where I can find rows which cause the UNDEFS

my heart skips a bit every time i see undef, i can't stand it..

**Comment by vvitad** [comment]
@bersace 
![image](https://user-images.githubusercontent.com/22113207/234550855-9856d71f-4708-41e7-9d29-f80aa82a0a0f.png)
Hi! Could you please give comment if this is normal behavior and what is this number, bytes, kilobytes? doest dot matter?

**Comment by bersace** [comment]
#1213 is merged and 8.1 will fix this. Thanks @vvitad for the feedback.
